client/src/components/HtmlEditorField/HtmlEditorField.js

```diff
@@ -54,7 +54,7 @@ class HtmlEditorField extends TextField {
    */
   registerChangeListener() {
     const target = this.getEditorElement();
-    this.getEditor().on('change keyup setcontent', () => {
+    this.getEditor().on('change keyup', () => {
```

Comment:
This change was made to resolve the "Blocks get double saved - which turns them back to "modified" issue. The reason this was happening is that form-schema value of a blank HTMLText field (or Varchar field for that matter), starts of as `null`.

However as part of the HTMLEditorField initialising it has a `setcontent` call made to it, which in turns tells the redux that "this form has changes" and therefore `formDirty` is always set for the element, which is obviously wrong. `formDirty` is used in the new logic that determines if the element form should be inline submitted when clicking a page/parent save button.

I have regression tested in in sink using this [PR](https://github.com/silverstripe/silverstripe-admin/pull/1796) that only contains this change.
That [run of sink](https://github.com/creative-commoners/recipe-kitchen-sink/actions/runs/9983689111) has the same failures as a [vanilla sink run](https://github.com/silverstripe/recipe-kitchen-sink/actions/runs/9984678122)